chosen/chosen.jquery.js

```diff
@@ -127,6 +127,7 @@ Copyright (c) 2011 by Harvest
       this.results_showing = false;
       this.result_highlighted = null;
       this.result_single_selected = null;
+      this.enable_group_select = this.options.enable_group_select || false;
```

Comment:
Watch out! You've got the logic backwards here. Should actually be:
```suggestion
      this.enable_group_select = this.options.enable_group_select || true;
```